.github/workflows/release-update-stable-tag.yml

```
@@ -0,0 +1,268 @@
+          echo "=========================="
+
+          # Update the stable tag
+          sed -i "s/^Stable tag: .*/Stable tag: $TARGET_VERSION/" "$README_PATH"
+
+          echo "📝 Updated stable tag line:"
+          grep "^Stable tag:" "$README_PATH" || echo "::error::Stable tag line not found"
+          echo "=========================="
+
+          # Check if there are any changes
+          if svn diff --quiet; then
+            echo "ℹ️ No changes detected - stable tag is already $TARGET_VERSION"
+          else
+            echo "📤 Changes detected. Committing changes to SVN..."
+            svn commit -m "Setting stable tag to $TARGET_VERSION" --username "${{ secrets.WPORG_SVN_USERNAME }}" --password "${{ secrets.WPORG_SVN_PASSWORD }}" --non-interactive
+            echo "✅ Successfully committed stable tag update to $TARGET_VERSION"
+          fi
+
+  update-gh-stable-tag:
+    runs-on: ubuntu-latest
+    needs: [validate-release, update-wporg-stable-tag]
+    strategy:
+      matrix:
+        branch:
+          - trunk
+          - ${{ needs.validate-release.outputs.branch }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          sparse-checkout: |
+            plugins/woocommerce/readme.txt
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Configure Git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
+      - name: Update stable tag on ${{ matrix.branch }}
+        env:
+          TARGET_VERSION: ${{ needs.validate-release.outputs.version }}
+          BRANCH_NAME: ${{ matrix.branch }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "🔄 Processing branch: $BRANCH_NAME..."
+
+          git checkout "$BRANCH_NAME"
+
+          README_PATH="plugins/woocommerce/readme.txt"
+
+          if [ ! -f "$README_PATH" ]; then
+            echo "::error::readme.txt not found at $README_PATH"
+            exit 1
+          fi
+
+          echo "📋 Current stable tag line on $BRANCH_NAME:"
+          grep "^Stable tag:" "$README_PATH" || echo "::error::Stable tag line not found"
+          echo "=========================="
+
+          # Update the stable tag
+          sed -i "s/^Stable tag: .*/Stable tag: $TARGET_VERSION/" "$README_PATH"
+
+          echo "📝 Updated stable tag line on $BRANCH_NAME:"
+          grep "^Stable tag:" "$README_PATH" || echo "::error::Stable tag line not found"
+          echo "=========================="
+
+          # Check if file was actually modified
+          if git diff --quiet "$README_PATH"; then
+            echo "ℹ️ No changes needed on $BRANCH_NAME - stable tag is already $TARGET_VERSION"
+          else
+            echo "📤 Changes detected. Creating PR for $BRANCH_NAME..."
+
+            UPDATE_BRANCH="update/stable-tag-$TARGET_VERSION-$BRANCH_NAME"
```

I'd suggest we add the timestamp to the branch name so it doesn't conflict with an existing branch that maybe wasn't already deleted.

```suggestion
            UPDATE_BRANCH="update/stable-tag-$TARGET_VERSION-$BRANCH_NAME-$(date +%s)"
```